### DISCUSSION TITLE
Chamilo Installation

### DISCUSSION THREAD

**Discussion by IsaacDBerry** [chamilo/chamilo-lms#5399] [General]
I am Installing Chamilo on a local server and when I get to the step 3 license step both the next and previous buttons stop working entirely preventing me from finishing the installation. Is there a fix for this?

**Comment by Kaneda-1** [comment]
Hi @IsaacDBerry what version of PHP do you use?

**Reply by IsaacDBerry** [reply]
I am using PHP version 8.2

**Comment by Kaneda-1** [comment]
Hi. 

You must use PHP 7.4.x with Chamilo 1.11.x.

Check:  
https://github.com/chamilo/chamilo-lms/releases

Although we do not officially support PHP 8.0, 8.1 or 8.2 with the 1.11 branch of Chamilo, we tested it relatively thoroughly with the -php74 package and it worked well, to the minor exception of the edition of learning paths which is visually a bit broken.

**We recommend using PHP 7.4** while we are running to deliver Chamilo 2.0 with full support for PHP 8.2 or superior.
